Eastern-Research-Group/mywaterway
# Feature/483 updated grid styles

## Related Issues:
* [HMW-483](https://jira.epa.gov/browse/HMW-483)

## Main Changes:
* Changing the column sizes of grids in HMW so they are more equally sized.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Verify the grids look good across all tabs 
    * Pay special attention to "Construction Stormwater" and "Industrial Stormwater" on discharger tabs
